src/connectors/planetradio.ts

```diff
@@ -1,26 +1,12 @@
 export {};
 
-Connector.playerSelector = [
-	'#__next > div > div > div:first-child',
-	'.now-playing-wrapper',
-];
+Connector.playerSelector = '#__next > div > div > div:first-child';
 
-Connector.artistSelector = [
-	'#__next > div > div > div:first-child > div:nth-child(5) > div > div:nth-child(2) > p:nth-child(2)',
-	'.text-right .title',
-];
+Connector.artistSelector = '#playbar-play-button + div p';
 
-Connector.trackSelector = [
-	'#__next > div > div > div:first-child > div:nth-child(5) > div > div:nth-child(2) > p:nth-child(3)',
-	'.text-right .track',
-];
+Connector.trackSelector = '#playbar-play-button + div p:last-child';
 
-Connector.trackArtSelector = [
-	'#__next > div > div > div:first-child > div:nth-child(5) img',
-	'.now-playing-block.right .image',
-];
+Connector.trackArtSelector = '#playbar-play-button + div picture img';
 
-Connector.playButtonSelector = [
-	'#__next > div > div > div:first-child [data-icon="play"]',
-	'.play',
-];
+Connector.isPlaying = () =>
+	Util.getDataFromSelectors('#playbar-play-button svg', 'icon') !== 'play';
```

Comment:
```suggestion
Connector.playButtonSelector = '#playbar-play-button svg[data-icon="play"]';
```

This should work while maintaining declarative approach (untested).